### PR TITLE
Add config for aria labels and text labels, and fix test

### PR DESCRIPTION
**Pull request by thormeier**
Fixes https://github.com/ismail9k/vue3-carousel/issues/302 and implements https://github.com/ismail9k/vue3-carousel/issues/278.

I tried to stick to the code style as much as possible. I had to type the icons more explicitly for them to be translateable, but in turn, I also added translations for the icons. I added an example to the docs and added the new object to the config table. We might want to think of a better way to show the new config object since I couldn't indent it in the markdown table nicely.

Thank you very much for this library!

### CODE REVIEW

File: src/components/Pagination.ts [hunk 1]
```diff
@@ -25,7 +27,7 @@ const Pagination = () => {
         'carousel__pagination-button': true,
         'carousel__pagination-button--active': isActive(slide),
       },
-      'aria-label': `Navigate to slide ${slide + 1}`,
+      'aria-label': config.labels?.ariaNavigateToSlide + ' ' + (slide + 1).toString(),
```
Comment (thormeier) [inline src/components/Pagination.ts:30]: I used some kind of template to allow for simpler translations of `Item X of Y`, we perhaps want something similar here, too.

File: src/components/Icon.ts [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
   }
+
   const path = icons[iconName]
   const pathEl = h('path', { d: path })
 
-  const iconTitle = (props.title || iconName) as string
+  const iconTitle = (props.title
+    || (config.labels?.iconAriaLabels ? config!.labels!.iconAriaLabels![iconName]! as string : undefined)
+    || iconName
+  ) as string
```
Comment (thormeier) [inline src/components/Icon.ts:24]: Still uses the icon name as a fallback but prefers the configured title. Not sure if that's alright the way it is or if we should inject the translated aria label via props instead.

File: src/components/Pagination.ts (hunk 1 above)
Comment (ismail9k) [reply src/components/Pagination.ts:30]: Yes, It's better to have a unified API

**Comment by ismail9k** [comment]
@thormeier Thank you for your awesome work. :)

**Pull request merged**